frontend/packages/dev-console/src/components/pipelines/pipeline-form/PipelineResourceDropdownField.tsx

```diff
@@ -21,9 +21,7 @@ const PipelineResourceDropdownField: React.FC<PipelineResourceDropdownFieldProps
 }) => {
   const [field, { touched, error }] = useField(props.name);
   const [createMode, setCreateMode] = React.useState(false);
-  const { setFieldValue, setFieldTouched, setStatus, status, values } = useFormikContext<
-    FormikValues
-  >();
+  const { setFieldValue, setFieldTouched, setStatus, values } = useFormikContext<FormikValues>();
```

Comment:
If you're no longer reading from the `status` do you need `setStatus`? From what I can tell the only use of `setStatus` is in the `useRef` and that doesn't call upon it anymore.

Reply:
removed all the unused form helpers.